packages/components/tests/dummy/app/templates/components/side-nav.hbs

```diff
@@ -13,17 +13,32 @@
   <Shw::Text::H3>With generic content</Shw::Text::H3>
 
   <Shw::Flex as |SF|>
-    <SF.Item @label="Header + Body + Footer">
+    <SF.Item @label="Header + Body + Footer, base invocation">
+      <div class="shw-component-sim-side-nav-root-container">
+        <Hds::SideNav @isResponsive={{false}} @hasA11yRefocus={{false}}>
+          <:header>
+            <Shw::Placeholder @text="header" />
+          </:header>
+          <:body>
+            <Shw::Placeholder @text="body" />
+          </:body>
+          <:footer>
+            <Shw::Placeholder @text="footer" />
+          </:footer>
+        </Hds::SideNav>
+      </div>
+    </SF.Item>
+    <SF.Item @label="Header + Body + Footer, heights declared">
       <div class="shw-component-sim-side-nav-root-container">
         <Hds::SideNav @isResponsive={{false}} @hasA11yRefocus={{false}}>
           <:header>
-            <Shw::Placeholder @height="72px" @text="header" @background="#e4e4e4" />
+            <Shw::Placeholder @text="header" @height="72px" />
```

Comment:
@MelSumner I think there is a misunderstanding of the role of the height in the context of the SideNav elements, let's discuss when you have 5 minutes.